config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -47,6 +47,55 @@ presubmits:
       testgrid-dashboards: sig-testing-misc
       description: Runs conformance tests using kind and the conformance image
 
+  # conformance test using image against kubernetes master branch with `kind`
+  - name: pull-kubernetes-conformance-kind-ga-only
```

Comment:
copied `pull-kubernetes-conformance-image-test` as a starting point, not sure if there's a better one to start from

Reply:
I think that the recommendations is to move to podutils 
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml#L222-L268

Reply:
ah, ok, I was trying to set up a conformance run so I based it on the existing conformance one.